app/views/tmdb/actor_credit.html.erb

```diff
@@ -1,52 +1,43 @@
 <article id="profile">
```

Comment:
While this looks like a lot of changes here, they're really whitespace changes. I cleaned up some rogue closing tags  and adjusted the indenting.